.github/workflows/fetch_filter_resources.yaml

```
@@ -174,6 +174,9 @@ jobs:
       - name: Filter tools for communities
         run: | 
           bash sources/bin/get_community_tools.sh
+      - name: Populate Labs of communities
+        run: |
+          bash sources/bin/populate_labs.sh
```

populate_labs.sh ne définie pas la communauté, il faut le définir dans le yaml (ou changer le code du script)

```suggestion
          bash sources/bin/populate_labs.sh
        env: 
          COMMUNITY: biodiversity
```